src/render/render-webgpu.ts

```diff
@@ -273,5 +287,51 @@ export function createWebGPURenderer(
     return gpuBuffer.getMappedRange();
   }
 
+  async function stats(): Promise<Map<string, bigint>> {
+    const byteSize = (MAX_PIPELINES + 1) * 8;
+    // We need to have 2 buffers here because you can't have MAP_READ
+    // and QUERY_RESOLVE on the same buffer. We resolve the QuerySet
+    // to one buffer, then copy it to another for reading.
+    const resolveBuffer = device.createBuffer({
+      // GPUSize64s
+      size: byteSize,
+      usage: GPUBufferUsage.QUERY_RESOLVE | GPUBufferUsage.COPY_SRC,
+      mappedAtCreation: false, // mapped post texture copy
+    });
+    const copyBuffer = device.createBuffer({
+      size: byteSize,
+      usage: GPUBufferUsage.COPY_DST | GPUBufferUsage.MAP_READ,
+      mappedAtCreation: false, // mapped post texture copy
+    });
```

Comment:
future todo: Daryl would like to massage this a bit so that we use the CY system to create a set of static buffers so we're not allocating buffers every frame and so all buffer creation goes through CY